src/shapes/IText/IText.ts

```diff
@@ -497,10 +497,14 @@ export class IText<
       lineIndex = cursorLocation.lineIndex,
       charIndex =
         cursorLocation.charIndex > 0 ? cursorLocation.charIndex - 1 : 0,
-      charHeight = this.getValueOfPropertyAt(lineIndex, charIndex, 'fontSize'),
+      charHeight = this.getValueOfPropertyAt<'fontSize'>(
```

Comment:
Before it gets asked, i tried to derive the type from the third argument right away from inside the function  `getValueOfPropertyAt` , but it does not work.
This return string/number reliably so far.